### PR TITLE
[14.0][MIG] account_invoice_validation_queued

### PR DESCRIPTION
**Pull request by iwkse**

### CODE REVIEW

**Comment by iwkse** [comment]
@pedrobaeza ready

File: account_invoice_validation_queued/migrations/14.0.2.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if not openupgrade.table_exists(env.cr, "account_move_queue_job_rel"):
+        return
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO account_move_validation_job_rel
```
Comment (pedrobaeza) [inline account_invoice_validation_queued/migrations/14.0.2.0.0/post-migration.py]: Why is this needed? Can't you preserve the old name or rename the table in pre?

Comment (iwkse) [reply account_invoice_validation_queued/migrations/14.0.2.0.0/post-migration.py]: @pedrobaeza means like that?

Comment (pedrobaeza) [reply account_invoice_validation_queued/migrations/14.0.2.0.0/post-migration.py]: Not really. Now I see both post and pre migration scripts doing the insert. My question about why is this needed still unanswered, and without it, I can't recommend you the actions to take.

Comment (iwkse) [reply account_invoice_validation_queued/migrations/14.0.2.0.0/post-migration.py]: Right. That's something from 13.0 and I believe it can be removed. I've pushed without the migration folder.
Thank you @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Comment by iwkse** [comment]
@pedrobaeza There's the README which should be updated as it doesn't reflect the actual steps to see the validation popup.
I've seen the information seems to be wrong also for the 13.0, so I'm a bit confused to understand how to find such popup.
Is such documentation valid for 12.0?
Could you please help to clarify it?
Thank you!

**Comment by pedrobaeza** [comment]
The action entry is called "Post entries" instead.

**Comment by iwkse** [comment]
@pedrobaeza README updated!

**Review by francesco-ooops — APPROVED** [review]
From "usage":

If any of the selected invoices are not in draft state, you will receive an error.

This is not working - it is't working on 13 either, while it's ok on 12. I suggest to change description as the feature doesn't seem so important (posted invoices do not get enqueued anyway)

For the rest, functional review ok for what job queue allows

**Comment by iwkse** [comment]
> From "usage":
> 
> If any of the selected invoices are not in draft state, you will receive an error.
> 
> This is not working - it is't working on 13 either, while it's ok on 12. I suggest to change description as the feature doesn't seem so important (posted invoices do not get enqueued anyway)
> 
> For the rest, functional review ok for what job queue allows

@pedrobaeza what do you think?

**Comment by pedrobaeza** [comment]
Proceed the way you think is better.

**Comment by iwkse** [comment]
@pedrobaeza PR updated with changes in USAGE.rst

**Comment by iwkse** [comment]
@pedrobaeza could you try now merging?

thank you
